design/ceph/object/zone.md

```diff
@@ -36,6 +38,8 @@ spec:
     erasureCoded:
       dataChunks: 6
       codingChunks: 2
+  # customEndpoints:
```

Comment:
In the docs let's uncomment this since it's recommended to set and the default is not useful.